spacy/schemas.py

```diff
@@ -218,6 +218,8 @@ class ConfigSchemaNlp(BaseModel):
     tokenizer: Callable = Field(..., title="The tokenizer to use")
     lemmatizer: Callable = Field(..., title="The lemmatizer to use")
     load_vocab_data: StrictBool = Field(..., title="Whether to load additional vocab data from spacy-lookups-data")
+    before_init: Optional[Callable] = Field(..., title="Optional callback to modify Language class before initialization")
+    after_init: Optional[Callable] = Field(..., title="Optional callback to modify nlp object after initialization")
```

Comment:
How would you feel about allowing these to default to `None`? The main disadvantage I can see is that we don't have much else that defaults to `None` here, so it'll be harder to document consistently.


```suggestion
    before_init: Optional[Callable] = Field(None, title="Optional callback to modify Language class before initialization")
    after_init: Optional[Callable] = Field(None, title="Optional callback to modify nlp object after initialization")
```

Reply:
Yeah, this would be how we'd do it if we decide to make some config fields optional. The values here currently already default to `None`, but if we set it to `None` here, it will be auto-filled and the config validation will never complain about it not being set... currently that's inconsistent but if we want more settings to behave like this, we should definitely consider it for `before_init` and `after_init` as well.

Reply:
Okay cool